vendor/engines/split_accounts/app/services/split_accounts/attribute_splitter.rb

```diff
@@ -14,6 +14,10 @@ def split(original_object, split_object, split)
       end
     end
 
+    def to_ary
```

Comment:
It's a complicated question. I wanted to be able to use `Array`, which now researching might also try `to_a`, which I think I would prefer. I'll test it out.